catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswSourceConfiguration.java

```diff
@@ -92,7 +103,11 @@ public String getPassword() {
     }
 
     public void setPassword(String password) {
-        this.password = password;
+        String updatedPassword = password;
+        if (encryptionService != null && !password.equals("")) {
```

Comment:
I still need a null check for this method in CswSourceConfiguration and the setPassword method in AbstractCswSource. This is because as I am deprecating the default constructors in those classes, so there is a possibility the encryption service could be null if there is some code calling those constructors.